package.json

```diff
@@ -43,6 +43,10 @@
         "cordova": ">=3.1.0"
       },
       "4.0.0": {
+        "cordova": ">=3.1.0",
```

Comment:
I think minimum Cordova version should be increased as well, both here and in plugin.xml. I don't think we want to keep supporting old Cordova versions:)

Reply:
See https://lists.apache.org/thread.html/rc277f523ceb625820acd4b361e08b3b0269108f19a4d5e85b9d104fd%40%3Cdev.cordova.apache.org%3E